<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>697</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>10</width>
    <height>10</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>899</width>
    <height>697</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Monitoring</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icons/app.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(233, 185, 110);
color: rgb(164, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridWidget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>1</y>
      <width>881</width>
      <height>651</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>1000</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item row="6" column="0" colspan="2">
        <widget class="QTextBrowser" name="textBrowser">
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="privKeyLineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(164, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="addressLineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(164, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="addressLabel">
         <property name="text">
          <string>Address         :     </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(252, 249, 249);
background-color: rgb(78, 154, 6);</string>
           </property>
           <property name="text">
            <string>Start Komodo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="getInfoButton">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(252, 249, 249);
background-color: rgb(78, 154, 6);</string>
           </property>
           <property name="text">
            <string>Get Info</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="getPubKeyButton">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(252, 249, 249);
background-color: rgb(78, 154, 6);</string>
           </property>
           <property name="text">
            <string>Get Public Key</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="getPrivKeyButton">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(252, 249, 249);
background-color: rgb(78, 154, 6);</string>
           </property>
           <property name="text">
            <string>Get Private Key</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="insertPrivKeyButton">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(252, 249, 249);
background-color: rgb(78, 154, 6);</string>
           </property>
           <property name="text">
            <string>Insert Private key</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="privKeyLabel">
         <property name="text">
          <string>Private Key   : </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="KillButton">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(252, 249, 249);
background-color: rgb(164, 0, 0);</string>
         </property>
         <property name="text">
          <string>kill Komodo</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2" alignment="Qt::AlignTop">
        <widget class="QTextBrowser" name="textBrowser2">
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionKomodo"/>
   <addaction name="separator"/>
   <addaction name="actionAgama"/>
   <addaction name="separator"/>
   <addaction name="actionBarterDEX"/>
   <addaction name="separator"/>
   <addaction name="actionCoinIssuer"/>
   <addaction name="actionMarketMaker"/>
   <addaction name="actionKomodo_CLI"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInstall"/>
   <addaction name="actionUninstall"/>
   <addaction name="actionCleanKomodo"/>
   <addaction name="actionGetInfo"/>
  </widget>
  <action name="actionAgama">
   <property name="icon">
    <iconset resource="kmdrsc.qrc">
     <normaloff>:/icons/agama_app_icon.ico</normaloff>:/icons/agama_app_icon.ico</iconset>
   </property>
   <property name="text">
    <string>Agama</string>
   </property>
   <property name="toolTip">
    <string>Open Agama log center</string>
   </property>
  </action>
  <action name="actionKomodo">
   <property name="icon">
    <iconset resource="kmdrsc.qrc">
     <normaloff>:/icons/komodo.png</normaloff>:/icons/komodo.png</iconset>
   </property>
   <property name="text">
    <string>Komodo</string>
   </property>
   <property name="toolTip">
    <string>Open komodo log center</string>
   </property>
  </action>
  <action name="actionBarterDEX">
   <property name="icon">
    <iconset resource="kmdrsc.qrc">
     <normaloff>:/icons/barterdex.ico</normaloff>:/icons/barterdex.ico</iconset>
   </property>
   <property name="text">
    <string>BarterDEX</string>
   </property>
   <property name="toolTip">
    <string>Open BarterDEX log center</string>
   </property>
  </action>
  <action name="actionJumblr">
   <property name="icon">
    <iconset resource="kmdrsc.qrc">
     <normaloff>:/icons/jumblr.png</normaloff>:/icons/jumblr.png</iconset>
   </property>
   <property name="text">
    <string>Jumblr</string>
   </property>
   <property name="toolTip">
    <string>Open Jumblr log center</string>
   </property>
  </action>
  <action name="actionCoinIssuer">
   <property name="icon">
    <iconset resource="kmdrsc.qrc">
     <normaloff>:/icons/Coin.png</normaloff>:/icons/Coin.png</iconset>
   </property>
   <property name="text">
    <string>CoinIssuer</string>
   </property>
   <property name="toolTip">
    <string>Create your own chain</string>
   </property>
  </action>
  <action name="actionInstall">
   <property name="icon">
    <iconset resource="kmdrsc.qrc">
     <normaloff>:/icons/install.png</normaloff>:/icons/install.png</iconset>
   </property>
   <property name="text">
    <string>Install</string>
   </property>
   <property name="toolTip">
    <string>download and install Komodo</string>
   </property>
  </action>
  <action name="actionUninstall">
   <property name="icon">
    <iconset resource="kmdrsc.qrc">
     <normaloff>:/icons/Uninstall.png</normaloff>:/icons/Uninstall.png</iconset>
   </property>
   <property name="text">
    <string>Uninstall</string>
   </property>
   <property name="toolTip">
    <string>Delete source code and uninstall komodo</string>
   </property>
  </action>
  <action name="actionCleanKomodo">
   <property name="icon">
    <iconset resource="kmdrsc.qrc">
     <normaloff>:/icons/clean.ico</normaloff>:/icons/clean.ico</iconset>
   </property>
   <property name="text">
    <string>CleanKomodo</string>
   </property>
   <property name="toolTip">
    <string>Clean Komodo and delete all blocks</string>
   </property>
  </action>
  <action name="actionGetInfo">
   <property name="icon">
    <iconset resource="kmdrsc.qrc">
     <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>GetInfo</string>
   </property>
   <property name="toolTip">
    <string>Get komodo information</string>
   </property>
  </action>
  <action name="actionKomodo_CLI">
   <property name="icon">
    <iconset resource="kmdrsc.qrc">
     <normaloff>:/icons/cli.png</normaloff>:/icons/cli.png</iconset>
   </property>
   <property name="text">
    <string>Komodo CLI</string>
   </property>
   <property name="toolTip">
    <string>Open komodo CLI</string>
   </property>
  </action>
  <action name="actionMarketMaker">
   <property name="icon">
    <iconset resource="kmdrsc.qrc">
     <normaloff>:/icons/mmcli.png</normaloff>:/icons/mmcli.png</iconset>
   </property>
   <property name="text">
    <string>MarketMaker</string>
   </property>
   <property name="toolTip">
    <string>Open Market Maker Gui</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="kmdrsc.qrc"/>
 </resources>
 <connections/>
</ui>
